src/lib/manifestationFactoryUtils.js

```diff
@@ -136,7 +136,16 @@ export function formatMaterialTypesToCode(materialTypeArray) {
 }
 
 //TODO check if compound material types are sorted
-export function constructMaterialType(materialTypes) {
+/**
+ *  
+    
+ * @param {Array<Object>} materialTypes [ 
+  {"materialTypeSpecific": {"display": "lydoptagelse (cd)","code": "SOUND_RECORDING_CD"},
+  {"materialTypeSpecific": {"display": "bog", "code": "BOOK"}}
+]
+ * @returns {String} "Bog / Lydoptagelse (cd)"
+ */
```

Comment:
Jeg retter også den her JSDoc, for IntellJ bliver sur. (Forklaringen er, at du har glemt at skrive `@example` omkring dit eksempel